app/renderer/ui/components/radonOperator/index.tsx

```diff
@@ -0,0 +1,184 @@
+import * as React from "react"
+
+import { getHashingFunctions, getMethodInformation, getMethodsByType, RadonType, getFilteringFunctions, getReducingFunctions } from "./radon"
+const styles = require("./style.scss")
+
+export interface RadonOperatorProps {
+  inputType: RadonType,
+}
+
+export interface RadonOperatorState {
+  outputType: RadonType,
+  availableMethodsNames: any,
+  selectedMethodInfo: any,
+  selectedFunctionFilter: any,
```

Comment:
Can you replace selectedFunctionFilter, selectedFunctionReduce and selectedFunctionHash to a single key?